### PR TITLE
Moving Sashimi integration tests to Calamari

### PR DESCRIPTION
**Pull request by IsaacCalligeros95**
As a part of dependency management work we're moving any tests that require the Calamari project in the Sashimi Tests to the Calamari portion.

### CODE REVIEW

File: source/Calamari.Tests/CommandsFixture.cs [hunk 1]
```diff
@@ -843,5 +834,73 @@ void IgnoreIfVersionIsNotInRange(string minimum, string? maximum = null)
                 Assert.Ignore($"Test ignored as terraform version is not between {minimumVersion} and {maximumVersion}");
             }
         }
+
```
Comment (IsaacCalligeros95) [inline source/Calamari.Tests/CommandsFixture.cs:837]: Note I should be able to bump the test project to net462 as we've done elsewhere and remove this code. We would need to include the Sashimi.Tests.Shared package reference though which isn't ideal...

File: source/Calamari.Tests/Calamari.Tests.csproj [hunk 2]
```diff
@@ -7,21 +7,48 @@
...
+        </None>
+        <None Update="GoogleCloud\example.tf">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Update="GoogleCloud\example.tfvars">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Update="GoogleCloud\test.txt">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
```
Comment (mik-ky) [inline source/Calamari.Tests/Calamari.Tests.csproj]: We can remove these as it's covered by the above globs right? `**/*.tf*` and `**/*.txt`

**Review by mik-ky — CHANGES_REQUESTED** [review]
LGTM, just a minor comment.

File: source/Calamari.Tests/Calamari.Tests.csproj (hunk 2 above)
Comment (ghost) [reply source/Calamari.Tests/Calamari.Tests.csproj]: Thanks for spotting~

**Review by mik-ky — APPROVED** [review]
Nice, LGTM.

**Pull request merged**